### DISCUSSION TITLE
Power/Energy Calculation

### DISCUSSION THREAD

**Discussion by edetbijoyk** [IBM/aihwkit#414] [General]
I would like to know how to calculate the power/energy consumption of a model (or any design that uses analog arrays for computing matrix multiplication) using aihwkit?  
Responses would be highly appreciated.
Thanks!

**Comment by maljoras** [comment]
Unfortunately, we currently do not support such calculations. The simulator is currently only considering the effect of analog non-idealities on accuracy.
